src/Tools/AI Test Toolkit/src/SampleTests/MarketingTextSample/MarketingTextQualityTest.Codeunit.al

```diff
@@ -0,0 +1,149 @@
+codeunit 149033 "Marketing Text Quality Test"
+{
+    Subtype = Test;
+    TestPermissions = Disabled;
+
+    var
+        AITestContext: Codeunit "AIT Test Context";
+        Context: Text;
+        Question: Text;
+        Format: Enum "Entity Text Format";
+        Tone: Enum "Entity Text Tone";
+
+    [Test] // Test No. 0
+    procedure CheckLengthOfMarketingText()
+    var
+        TestSetup: Codeunit "Test Input Json";
+        Answer: Text;
+        Facts: Dictionary of [Text, Text];
+    begin
+        // [Scenario] Test check the maximum length of the marketing text
+        // [GIVEN] Item with attributes
+        // Sample from the dataset:
+        // {"test_setup": {"Product Name":"ATHENS Mobile Pedestal","Color":"Black","Depth":"75 CM","Height":"90 CM","Material Description":"Wood","Model Year":"1942","Category":"Assorted Tables"}}
+        TestSetup := AITestContext.GetTestSetupAsJson();
+        Format := Enum::"Entity Text Format"::TaglineParagraph;
+        Tone := Enum::"Entity Text Tone"::Inspiring;
+        Facts := ItemJsonPropertiesToDictionary(TestSetup);
+
+        // [WHEN] Generate marketing text with inspiring tone
+        Answer := GenerateMarketingText(Facts, Tone, Format);
+
+        // [THEN] Assert that the length of the marketing text is less than or equal to 1000 characters
+        if StrLen(Answer) > 1000 then
+            Error('The length of the marketing text is greater than 1000 characters. Actual length: %1', StrLen(Answer));
```

Comment:
Consider using the `Assert` Library codeunit instead of doing a manual error

Reply:
We will not ship the sample tests, they have been removed